components/Repository/RepositoryItemTopBar.tsx

```diff
@@ -30,6 +34,12 @@ export const RepositoryItemTopBar = ({
       <div className="flex flex-1 items-center justify-end">
         <div className={`h-2 w-2 rounded-full ${repositoryHasNewIssues ? "bg-primary" : ""}`}></div>
       </div>
+      {isIssueOpen && !isMedia720px && (
+        <RepositoryCollectivButton
```

Comment:
Will it work with GitLab repos?

Reply:
Will introduce a flow on our end for handling gitlab repos! Thanks.
@tiwarishankar please handle gitlab redirects it on Collectiv.

Reply:
yes added here for gitlab repo!!